pkgs/development/ocaml-modules/ocamline/default.nix

```diff
@@ -0,0 +1,22 @@
+{ ocamlPackages, fetchFromGitHub, lib }:
+
+ocamlPackages.buildDunePackage rec {
```

Comment:
Please take `buildDunePackage` and `linenoise` as individual arguments instead of the whole `ocamlPackages` set.